caluma-demo/nginx.conf

```diff
@@ -0,0 +1,41 @@
+server {
+  listen 80 default_server;
+  listen [::]:80 default_server;
+
+  root /usr/share/nginx/html;
+
+  server_name _;
+
+  proxy_set_header Host $http_host;
+
+  gzip_static on;
+  gzip_proxied any;
+
+  location / {
+    add_header Cache-Control 'private, max-age=86400';
+    try_files $uri $uri/ /index.html?/$request_uri;
+  }
+
+  location ~ ^/graphql {
```

Comment:
In our other projects we add a trailing slash to the location regex. I think this makes sense to make sure the intended path (in this case `/graphql/`) is matched and not a random one (for example `/graphqli/`).

Reply:
I wrote in the environment.js file "/graphql". That was the reason why I didn't add a slash in the end in the nginx.conf. Now it's fixed.